app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -44,6 +46,42 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        if (routeParams == null) {
+            return true;
+        }
+
+        MenuItem item = menu.findItem(R.id.action_edit);
+
+        if (item == null) {
+            return true;
+        }
+
+        String mode = "";
+        JSONObject session;
+        String creator = "";
+        String user = "";
+
+        try {
+            mode = routeParams.get("mode").toString();
+            session = new JSONObject(routeParams.getString("session"));
+
+            user = session.getJSONObject("user").get("id").toString();
+            creator = routeParams.getString("user");
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
+
+        if (mode.equals("play") && creator.equals(user) && !creator.equals("")) {
+            item.setVisible(true);
+        } else if (mode.equals("play")) {
+            item.setVisible(false);
```

Comment:
You don't need to explicitly `setVisible(false)` as it is invisible by default as per your settings in `menu_play.xml`